dictionaries/python/src/data_science.txt

```diff
@@ -94,3 +94,27 @@ upsample
 parallelizable
 ONNX
 enwik
+Hadamard
+Schur product
+rsqrt
+bucketizes
+tera
+strided
+invvar
+fastai
+behaviour
```

Comment:
This is the British spelling of `behavior`. Is there a need to add it here instead of enabling the British dictionary?

Reply:
you're right , revert done